core/src/main/resources/error/error-classes.json

```diff
@@ -536,12 +563,71 @@
       "Failed to set original permission <permission> back to the created path: <path>. Exception: <message>"
     ]
   },
+  "ROUTINE_ALREADY_EXISTS" : {
+    "message" : [
+      "Cannot create the function <routineName> because it already exists.",
+      "Choose a different name, drop or replace the existing function, or add the IF NOT EXISTS clause to tolerate a pre-existing function."
+    ],
+    "sqlState" : "42000"
+  },
+  "ROUTINE_NOT_FOUND" : {
```

Comment:
Why do you name it `ROUTINE...` if SQL key word is `FUNCTION...`. The last one is even used in the error message. Could you rename to `FUNCTION...` for consistency, otherwise, please, explain the difference.

Reply:
Future proofing: Eventually we will add procedures.  The shared name for both is routine. This is also the name in the SQL Standard information schema.